src/chat/dtos/requests/send-message.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, MinLength, MaxLength } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class SendMessageDto {
+  @ApiProperty({
+    description: '채팅 메시지 (1~50자)',
+    example: '안녕하세요!',
+    minLength: 1,
+    maxLength: 50,
+  })
+  @IsString()
+  @MinLength(1)
+  @MaxLength(50)
+  message: string;
```

Comment:
```suggestion
  @IsString()
  @MinLength(1)
  @MaxLength(50)
  @Transform(({ value }) => sanitizeHtml(value, {
    allowedTags: [], 
    allowedAttributes: {}
  }).trim())
  message: string;
```
이런식으로 service에서 validator를 하는게 아니라 dto로 들어올 때부터 하는건 어떨까요?